mephisto/abstractions/blueprints/static_html_task/source/dev/hooks.js

```diff
@@ -0,0 +1,33 @@
+import { useState, useCallback, useEffect } from "react";
+
+/**
+ * This hook is to be used with events that are supposed to update state when they are consumed.
+ * @param {string} eventName
```

Comment:
small nit: I would rename `eventName` to `configName`.

I consider `HIDE_SUBMIT_BUTTON` to be a config property, not an event